packages/esm-patient-labs-app/src/test-results/individual-results-table/individual-results-table.component.tsx

```diff
@@ -94,7 +94,7 @@ const IndividualResultsTable = ({ isLoading, parent, subRows, index }) => {
           </span>
         ),
         value: {
-          value: (row.obs[0]?.value ? row.obs[0]?.value : '') + +' ' + (row?.units ? ` ${row?.units}` : ''),
+          value: (row.obs[0]?.value ? row.obs[0]?.value : '') + ' ' + (row?.units ? ` ${row?.units}` : ''),
```

Comment:
```suggestion
          value: `${row.obs[0]?.value ?? ''} ${row.units ?? ''}`,
```

Reply:
This would work the same way, and is hopefully easier to reason about, and less likely to mess up.

Reply:
> 

Thanks @denniskigen .Done